common/djangoapps/course_modes/views.py

```diff
@@ -148,6 +150,20 @@ def get(self, request, course_id, error=None):
             "responsive": True,
             "nav_hidden": True,
         }
+
+        enterprise_learner_data = enterprise_api.get_enterprise_learner_data(site=request.site, user=request.user)
+        if enterprise_learner_data:
+            organizations = organization_api.get_course_organizations(course_id=course.id)
+            context["show_enterprise_context"] = True
+            context["partner_names"] = partner_name = course.display_organization \
```

Comment:
@mattdrayer I was actually using the partner_name as default on line 162. If we found the organizations data then simply overriding the `partner_names` context